projects/baby_driver/scripts/test_gpio_get.py

```diff
@@ -0,0 +1,120 @@
+"""This Module Tests methods in gpio_get.py"""
+# pylint: disable=unused-import
+import unittest
+from unittest.mock import patch
+
+from gpio_get import gpio_get
+from gpio_port import GpioPort
+from can_util import Message
+from message_defs import BABYDRIVER_DEVICE_ID, BABYDRIVER_CAN_MESSAGE_ID
+
+class TestGpioGet(unittest.TestCase):
+    """Test Babydriver's gpio_get function"""
+
+    @patch('can_util.send_message')
+    @patch('can_util.next_message')
+    def test_low_bound(self, mock_next_message, mock_send_message):
+        '''Tests minimum values for port and pin'''
+        gpio_pin_data = [1]
+        gpio_pin_msg = Message(
+            message_id=BABYDRIVER_CAN_MESSAGE_ID,
+            device_id=BABYDRIVER_DEVICE_ID,
+            data=gpio_pin_data,
+        )
+
+        status_data = [0]
+        status_msg = Message(
```

Comment:
This data lists need to include the babydriver ID as the first element, since that's the protocol. I'd also prefer that you use `can_pack` to pack the data even though it's technically unnecessary. (Also applies to the other message data lists below.)